.ci/benchmarks.groovy

```diff
@@ -9,11 +9,16 @@ pipeline {
     AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
     EC_KEY_SECRET = 'secret/observability-team/ci/elastic-cloud/observability-pro'
     BENCHMARK_ES_SECRET = 'secret/apm-team/ci/benchmark-cloud'
+    KIBANA_SECRET = 'secret/k8s/elastic-apps/oauth-proxy/prod/domains/observability-benchmarks.elastic.dev'
```

Comment:
What do you think if this secret is not exposed but a new one is created in `secret/observability-team/ci`?
Maybe `secret/observability-team/ci/benchmark-cloud-reporting`, in addition a new elasticsearh user is created with the right permissions so we don't expose the superuser within the CI?